docker/docker-compose.yml

```diff
@@ -5,6 +5,10 @@ services:
       context: ..
       dockerfile: docker/Dockerfile
     command: /bin/bash -c "./docker-entrypoint.sh dotnet DfE.FindInformationAcademiesTrusts.dll"
+    develop:
+      watch: 
+        - path: ..
+          action: rebuild
```

Comment:
Never heard of this feature but reading up on it, it will simply rebuild the container if local files in the parent folder change?

What benefit is there to this other than knowing that your docker image will build anytime you make a change?

Reply:
That's pretty much the whole point and given the build time improvements it's feasible to use ;-) 
It means that someone writing UI tests against a branch can add a test id to an element, save the file and not have to worry about how to build the application or when to rebuild it